data.json

```diff
@@ -2,7 +2,7 @@
   {
     "taishanese": "洗面",
     "taishaneseAudio": "http://www.stephen-li.com/TaishaneseVocabulary/Grooming/Audio/grooming%20Part%2001%20of%2048.mp3",
-    "taishaneseRomanization": "[ɬai55 mein32]",
+    "taishaneseRomanization": "[ɬai55 mein32]",
```

Comment:
Oh seems like you changed a non-breaking space into a regular space. I can probably take care of this in an automated way in the data extraction repo.